mirrord/layer/src/debugger_ports.rs

```diff
@@ -175,14 +241,16 @@ impl DebuggerType {
                 None
             }
         })
+        .collect::<Vec<_>>()
     }
 }
 
 /// Local ports used by the debugger running the process.
 /// These should be ignored by the layer.
 #[derive(Debug)]
 pub enum DebuggerPorts {
-    Detected(u16),
+    Detected(Vec<u16>),
+    Multiple(Vec<u16>),
```

Comment:
Why do we need two variants now? :eyes: 
Shouldn't it be just `Detected(Vec<u16>)`?